src/snowflake/snowpark/functions.py

```diff
@@ -3,6 +3,7 @@
 # Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
 #
 
+
```

Comment:
was this added by linter? If not, please remove the extra line

Reply:
I think saw. Well remove and if linter does not added again will leave as is.